aws/required-permissions.md

```diff
@@ -1,6 +1,6 @@
 # User permissions for deployment
 
-Users need to have specific permissions to execute the deployment. The file [deployment-role-policy.json](deployment-role-policy.json) contains the specific permissions that the users need to deploy the cloudformations templates. You can use a broad, permissive policy such as `AdministratorAccess`, and the deployment will work, but we recommend that roles are restricted to only the permissions required.
+Users need to have specific permissions to execute the deployment, update, and delete. The file [deployment-role-policy.json](deployment-role-policy.json) and [deployment-role-policy-for-account-scanner.json](deployment-role-policy-for-account-scanner.json) contain the specific permissions that the users need to deploy the CloudFormation templates. You can use a broad, permissive policy such as `AdministratorAccess` and the deployment will work, but we recommend restricting roles to only the permissions required.
```

Comment:
```suggestion
Users need to have specific permissions to deploy, update, and delete. The file [deployment-role-policy.json](deployment-role-policy.json) and [deployment-role-policy-for-account-scanner.json](deployment-role-policy-for-account-scanner.json) contain the specific permissions that users need to deploy the CloudFormation templates. We recommend restricting roles to only the permissions required rather than a broad, permissive policy such as `AdministratorAccess` .
```